Pack display name validation (#1122)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Added the **doc-review** command to check spelling in .md and .yml files as well as a basic release notes review.

> * Added a validation that pack display name did not exist in our repo.

Reviewer: ```suggestion
* Added a validation that a pack's display name does not already exist in content repository.
```

Author: done
